vision/product-search/cloud-client/src/main/java/vision/snippets/PurgeProducts.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package vision.snippets;
+
+// [START vision_product_search_purge_orphan_products]
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.vision.v1.LocationName;
+import com.google.cloud.vision.v1.ProductSearchClient;
+import com.google.cloud.vision.v1.PurgeProductsRequest;
+
+import java.util.concurrent.TimeUnit;
+
+
+public class PurgeProducts {
+
+  /**
+   * Delete the product and all its reference images.
+   *
+   * @param projectId - Id of the project.
```

Comment:
I'm not sure we use the @param and @throws style javadocs.

Reply:
I will remove javadocs header and keep as regular comment 
`//Delete the product and all its reference images.`